puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/tfm-ror51-pr-test.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- job:
+    name: tfm-ror51-pr-test
+    project-type: workflow
+    logrotate:
```

Comment:
you configure `buildDiscarder` in he groovy file, so I'd think this is not needed here anymore?